source/changelog.html.md

```diff
@@ -0,0 +1,20 @@
+# Welkin API Change Log
+
+This document outlines the changes that Welkin has made to our external API.
+This document was started in June 2019 and covers all changes made since that date.
+
+**API Documentation URL:** [https://developers.welkinhealth.com](https://developers.welkinhealth.com)
+
+**Welkin's API is in active development.** We will be making backwards compatible changes over the coming months with little or no advanced notice. You should expect Welkin to add additional optional fields to resource schemas and new resource types to the API. Your integration should be built with this understanding and should not break with the addition of new fields or resource types. Use of strict schema validation is not recommended. Breaking changes will be communicated in advance of rollout and we will work with you to make the transition.
+
+# API Updates
+Updates are listed by the day the api or documentation was released.
+
+## June 18th, 2019
+* **[added]** Worker `Create` and `Update` methods. You can now create and update worker accounts via Welkin APIs.
```

Comment:
needs to be white-labeled by provider. Can it read something like "You can now opt in to create..." or "Enabled clients can now create..."

Reply:
thanks for pointing this out